install/install.py

```diff
@@ -62,7 +62,8 @@ def check_sudo():
         if sudo == 0:
             return
         else:
-            print("[WARNING] Your user does not have sudo priveleges. Some OWTF components require sudo permissions to install")
+            Colorizer.warning("[!] Your user does not have sudo privileges. "
```

Comment:
You could almost inline that or have it on two lines like
```python
Colorizer.warning(
    "[!] Your user does not have sudo privileges. Some OWTF components require sudo permissions to install")
 ```

Yes it's 121 characters on one line but for 1 character, it's acceptable.